python/lsst/ctrl/mpexec/cli/cmd/commands.py

```diff
@@ -124,8 +126,33 @@ def build(ctx: click.Context, **kwargs: Any) -> None:
     _unhandledShow(show, "build")
 
 
+def start_coverage(coverage_packages: Tuple) -> Any:
```

Comment:
Can we annotate this to return a `coverage.Coverage` object? I think the `coverage_packages` type annotation can be `tuple[str, ...]`

Since this is an internal function maybe call it `_start_coverage` and `_stop_coverage` (if they are public we should document them).